### PR TITLE
Add some documentation about the crawlers

### PR DESCRIPTION
**Pull request by cmadjar**
This adds some documentation about the conp-dataset crawlers for OSF and Zenodo.

@glatard if you could review the document I wrote to ensure everything has been correctly captured. I am sure there is room for improvement :). Thank you!

### CODE REVIEW

File: Developers-Notes/conp-dataset_crawlers.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# conp-dataset crawlers
+
+The conp-dataset crawler scripts are located under the `scripts` directory of the [conp-dataset](https://github.com/CONP-PCNO/conp-dataset) repository.
+
+## Available crawlers
+
+There are crawlers available for the following platforms:
+
+- Zenodo (https://zenodo.org)
+- OSF (https://osf.io/)
```
Comment (cmadjar) [inline Developers-Notes/conp-dataset_crawlers.md:10]: I thought we had crawlers for FRDR but it was not merged and Giulia is gone... Her PR is failing Circle CI and has conflict with the master branch... https://github.com/CONP-PCNO/conp-dataset/pull/401

Comment (cmadjar) [reply Developers-Notes/conp-dataset_crawlers.md:10]: I guess this will bring another point: how do we upgrade the crawlers in the VM? @glatard ?

**Comment by cmadjar** [comment]
@glatard I am going to continue installing the crawlers on my computer and figuring out how they work so I can better document in this PR. I will let you know once I am done updating the document. 

I added the label in-progress to mention I am still working on it.

File: Developers-Notes/conp-dataset_crawlers.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+### Zenodo crawler
+
+The list of all the datasets that have been tagged with the `canadian-open-neuroscience-platform` can be found using the ` keywords:"canadian-open-neuroscience-platform"` search on [Zenodo](https://zenodo.org) or via the following link: https://zenodo.org/search?page=1&size=20&q=keywords:%22canadian-open-neuroscience-platform%22&sort=mostrecent
+
+## Install the crawler locally
+
+The crawler requires Python 3 and the following packages to be installed via `pip`:
+
+```bash
```
Comment (cmadjar) [inline Developers-Notes/conp-dataset_crawlers.md:133]: pip install -r requirements.txt

**Review by glatard — COMMENTED** [review]
Nice, thanks for doing this! See my comments inline. At some point we should mention how private datasets are handled, it's possible but convoluted with Zenodo, and it's almost possible with OSF (we should point to the PR and issue).

File: Developers-Notes/conp-dataset_crawlers.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+## Where are the crawlers run?
+
+The crawlers are being run every 15 minutes on the `zenodo-crawl.acelab.ca` VM under the username `jzhou`.
+
+```bash
+ssh -p 4706 jzhou@zenodo-crawl.acelab.ca
+```
+
+To obtain the password, contact cecile.madjar@mcin.ca or tglatard@encs.concordia.ca.
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply Developers-Notes/conp-dataset_crawlers.md:20]: OK. Once I get a better idea on how everything works together, I will take care of that.

[the remark this answers is not in the document]
Comment (cmadjar) [reply Developers-Notes/conp-dataset_crawlers.md:20]: ticket created for the sys admins for that: https://redmine.cbrain.mcgill.ca/issues/18303

File: Developers-Notes/conp-dataset_crawlers.md (hunk 1 above)
Comment (cmadjar) [reply Developers-Notes/conp-dataset_crawlers.md:10]: I have a feeling I might be that someone once I figure it all out ;).

**Comment by cmadjar** [comment]
@glatard @emmetaobrien any chance you could review this next week? Thank you!

**Pull request merged**